src/main/resources/lang/en_US.json

```diff
@@ -289,7 +289,9 @@
     "copyModeSet": "The developer copy mode has been set to: %mode%. Press \"%keyName%\" to trigger a data copy!",
     "chroma": "Chroma",
     "worldRenderedCooldownTime": "§c[SBA]: §rCooldown: %time%s",
-    "entityOutlinesRequirement": "Requires Shaders, Fast Render, and Antialiasing to be OFF"
+    "entityOutlinesRequirement": "Requires Shaders, Fast Render, and Antialiasing to be OFF",
+    "version": "Version",
+    "build": "Build"
```

Comment:
Build is a really hard term to localize, what will this represent exactly?

Reply:
This will represent the build number part of the mod's version as shown in the first two screenshots. It's going to be shown like "Version v1.6.1 Build 1,500" or "Build 1,500". Should I change the keys to "modVersion" and "modBuild" or something more descriptive?

Reply:
What I mean is that the build number will probably not be that high so why would it even matter to begin with? Can’t we just leave it out?

Reply:
I decided to add it so it's easier to diagnose bugs for pre-release versions since I'll know exactly what commit the version a person is using is based on. I think it'll be a useful thing to have.

Reply:
But I feel like when a user copies the version you’d want that text to be in English so you could probably just display the version (and have the pre-release number in it, which would help you figure out the commit if you use tags here on github) and only if they copy would they get the build number. Would the build number be a commit hash btw?

Reply:
I think that's a good idea. I'll edit it and post screenshots in a bit. The build number is the Github Actions run number.

Reply:
Ok I removed the build display at the bottom and changed it to add build number only when copying (it's the +2). How does this look? The build string will be removed so it'll only be the version string.
![image](https://user-images.githubusercontent.com/22475143/150501029-d2ec3a3d-8e5a-4c64-9b3f-150fadd48f1c.png)
![image](https://user-images.githubusercontent.com/22475143/150501153-b888b920-44c1-4a4c-adb3-201cf573c761.png)